mappings/net/minecraft/menu/Menu.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS net/minecraft/class_1703 net/minecraft/menu/Menu
+	COMMENT A menu is an object used for synchronizing GUIs between the server and the client.
+	COMMENT
+	COMMENT <p>Menus are usually used for syncing the GUIs of container blocks,
+	COMMENT and they have a list of {@linkplain Slot slots} that store items.
+	COMMENT The slots are usually controlled by the server.
+	COMMENT
+	COMMENT <p>Menus also contain a list of {@linkplain net.minecraft.menu.property.Property properties}
+	COMMENT that are used for syncing integers (e.g. progress bars) from the server to the client.
+	COMMENT
+	COMMENT <p>The {@link #syncId} is shared between the two sides. It is used to verify that a player
+	COMMENT has a menu open so that they can move items, for example.
+	COMMENT
+	COMMENT <p>On the client, menus are visualized by a {@link net.minecraft.client.gui.screen.ingame.MenuScreen}.
```

Comment:
maybe mention it has a reference to a client menu?

Reply:
Sure.